lib/constants/error_messages.js

```diff
@@ -24,6 +24,7 @@ exports[403004] = 'Invalid custom JSON parser. The specified value must be a fun
 exports[403005] = 'Invalid custom XML parser. The specified value must be a function.';
 exports[403006] = 'Invalid keep alive value. The specified value must be a boolean.';
 exports[403007] = 'Invalid custom credential manager value. The specified value must be an object, and it should have three methods: write, read, remove';
+exports[403008] = 'Invalid proxy value. The specified value must be a boolean.';
```

Comment:
Change the parameter name as a follow-up to the first comment.

Reply:
Replaced it with useEnvProxy.